File: Homework_1/CPP/10.cpp
#include <iostream>

int main()
{
    int a,b;
    char x;
    std::cout << "Enter a number: ";
    std::cin >> a;
    std::cout << "Enter an operation: ";
    std::cin >> x;
    std::cout << "Enter another number: ";
    std::cin >> b;

    if (x == '+')
        std::cout << a + b << std::endl;
    else if (x == '-')
        std::cout << a - b << std::endl;
    else if (x == '*')
        std::cout << a * b << std::endl;
    else if (x == '/')
        std::cout << a / b << std::endl;
    else if (x == '%')
        std::cout << a % b << std::endl;
    else
        std::cout << "Unknown operator\n";
    return 0;
}

File: Homework_1/JS/10.js
const readline = require('readline-sync');
let val1 = readline.questionInt("Enter a number: ");
let op = readline.question("Enter an operation: ");
let val2 = readline.questionInt("Enter another number: ");

if (op === '+')
    console.log(val1 + val2);
else if (op === '-')
    console.log(val1 - val2);
else if (op === '*')
    console.log(val1 *val2);
else if (op === '/')
    console.log(val1 / val2);
else if (op === '%')
    console.log(val1 % val2);
else
    console.log("Unknown operation");